ElevenLabs/Packages/com.rest.elevenlabs/Runtime/Models/PartialTranscribedAudioClip.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using UnityEngine;
+
+namespace ElevenLabs.Models {
+	/// <summary>
+	/// Represents timing information for a single character in the transcript
+	/// </summary>
+	[Serializable]
+	public class PartialTranscribedAudioClip
+	{
+		public readonly AudioClip AudioClip;
+		public readonly TimestampedTranscriptCharacter[] timestampedTranscriptCharacters;
```

Comment:
I think bc this is serializable it'll be better to have them not be readonly, be private and have property accessors.